Makefile

```diff
@@ -37,6 +37,9 @@ ios:
 test: all
 	build/env.sh go run build/ci.go test
 
+lint: ## Run linters. Use make install-linters first.
+	build/env.sh go run build/ci.go lint
```

Comment:
There's no `install-linters` step.

Reply:
Oh wait, it's not removed? It didn't show up for me until now.